src/styles/constants/_media-queries.scss

```diff
@@ -1,10 +1,12 @@
 $size-small-max: 374px;
+$size-small-medium-max: 460px;
```

Comment:
Is this how we usually handle variable names for in-between values?

Reply:
I searched the code for in-between values for media queries and actually didn't find any. I'm open to suggestions, but "small-medium" follows the same logic as a "medium-large" size (ie between medium and large).

Reply:
Yeah that makes sense. I would defer to @jstoffan if we want to follow any specific naming convention but otherwise looks good to me!